README.md

```diff
@@ -34,9 +34,10 @@ As blockchain technology proliferates, blockchain integration will become an inc
       -p 4200:4200 \
       ghcr.io/hyperledger/cactus-example-supply-chain-app:2021-09-08--docs-1312
     ```
-3. Wait for the output to show the message `INFO (api-server): Cactus Cockpit reachable http://0.0.0.0:3100`
-4. Visit http://localhost:3100 in a web browser with Javascript enabled
-5. Use the graphical user interface to create data on both ledgers and observe that a consistent view of the data from different ledgers is provided.
+3. Wait for the output to show the message `INFO (api-server): Cactus Cockpit reachable http://0.0.0.0:3200`
```

Comment:
@elenaizaguirre Reminder/note to self: Once the PR is finalized we need to do an extra step of publishing the new image and then updating the tag in here so that people can get the updated code you wrote (the old one won't have the prompt)

Reply:
@elenaizaguirre It's good to go now, so please publish the new image and then update here and also in the inner README of the supply chain app itself (best to do a free text search for the image tag project-wide to find all usages).
Once that's done, it's ready to merge IMO.

Reply:
@petermetz As we talked about [here](https://github.com/hyperledger/cactus/issues/1579#issuecomment-1022717133), I need the newest npm packages before create the image